Update BoilerplateProjects.md (#4593)
webdriverio/webdriverio: docs/BoilerplateProjects.md

## v5 Boilerplate Projects

### [amiya-pattnaik/webdriverIO-with-cucumberBDD](https://github.com/amiya-pattnaik/webdriverIO-with-cucumberBDD)

- Framework: Cucumber (v5.x), wdio-v5
- Features:

>     - [Page Object](PageObjects.md) Models used with ES6 style class-based approach, and full ES6–ES8 support through Babel

Reviewer: ```suggestion
    - [Page Object](PageObjects.md) Models used with ES6 style class-based approach and full ES6–ES8 support through Babel
```

>     - Examples of multi selector option to query elements with more than one selector at a time

Reviewer: ```suggestion
    - Examples of multi-selector option to query elements with more than one selector at a time
```

    - Examples of headless browser execution, using PhantomJS and Chrome
    - Integration with BrowserStack, SauceLabs

>     - Support of reading and writing from MS-Excel documentsfor easy test data management from external data sources with examples

Reviewer: ```suggestion
    - Support of reading and writing from MS-Excel documents for easy test data management from external data sources with examples
```

    - Database support for any RDBMS (Oracle, MySql, TeraData, Vertica etc.), executing queries, fetching result sets, and so on. Includes examples of E2E testing.
    - Multiple reporting (Spec, Junit, Allure, JSON), plus local view of JUnit report(`.html`) format
    - Task manage through Grunt

>     - Examples with demo app https://search.yahoo.com  and http://www.phptravels.net, Chai assertion liberary (`expect`, `assert`, `should`)

Reviewer: ```suggestion
    - Examples with demo app https://search.yahoo.com and http://www.phptravels.net, Chai assertion liberary (`expect`, `assert`, `should`)
```

>     - Appium-specific `.config` file for playback on mobile device. For one-click Appium setup, refer to: appium-setup-made-easy-OSX(https://github.com/amiya-pattnaik/appium-setup-made-easy-OSX)

Reviewer: ```suggestion
    - Appium-specific `.config` file for playback on mobile device. For one-click Appium setup, refer to: [appium-setup-made-easy-OSX](https://github.com/amiya-pattnaik/appium-setup-made-easy-OSX)
```
